README.md

```diff
@@ -177,10 +177,10 @@ Here are some simple schemes:
 
 Copyright (C) 2018 PeerTube Contributors
 
-This program is free software: you can redistribute it and/or modify
+This is a free software: you can redistribute and/or modify
```

Comment:
This is actually from the official GNU text: https://www.gnu.org/licenses/gpl-3.0.en.html

Reply:
Yes, of course. I should have realized it was from the official GNU website.
Thank you :)